devnet/tx-verifier-monitor/Dockerfile.cast

```diff
@@ -0,0 +1,13 @@
+# These versions are pinned to match the Dockerfile in the `ethereum/`
+# directory. Otherwise, there is nothing special about them and they can be
+# updated alongside the other Dockerfile.
+FROM ghcr.io/foundry-rs/foundry:nightly-55bf41564f605cae3ca4c95ac5d468b1f14447f9@sha256:8c15d322da81a6deaf827222e173f3f81c653136a3518d5eeb41250a0f2e17ea as foundry
```

Comment:
This does not work on mac without specifying `--platform=linux/amd64`. Please update (see `ethereum/Dockerfile`).

Reply:
Alternatively, I built a multi-platform one over at https://github.com/orgs/wormholelabs-xyz/packages/container/package/foundry, but I haven't had time to debug #4186 yet.